### PR TITLE
Meeting 2021-06-30.md

### PR DESCRIPTION
**Pull request by lucaslcode**

### CODE REVIEW

**Review by mrinal2412 — DISMISSED** [review]

File: docs/meetings/frontend/2021-06-30.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Frontend meeting 30 June 2021
+
+**Present:** Francisco Jesus Aapeli Lucas Darren
```
Comment (aapeliv) [inline docs/meetings/frontend/2021-06-30.md]: Want to sort this and add punctuation?

Comment (lucaslcode) [reply docs/meetings/frontend/2021-06-30.md]: It's in "Order of Lucas' zoom window"

Comment (aapeliv) [reply docs/meetings/frontend/2021-06-30.md]: Commas though?

Comment (lucaslcode) [reply docs/meetings/frontend/2021-06-30.md]: Hey wait a minute, you just always get to go first with your "aaaaaaaapeli"

Comment (aapeliv) [reply docs/meetings/frontend/2021-06-30.md]: Anyway, it doesn't matter

Comment (aapeliv) [reply docs/meetings/frontend/2021-06-30.md]: > Hey wait a minute, you just always get to go first with your "aaaaaaaapeli"

You've figured it out ;)

**Pull request merged**